package.json

```diff
@@ -10,15 +10,16 @@
   "version": "7.1.0",
   "author": "Beau Gunderson <beau@beaugunderson.com> (https://beaugunderson.com/)",
   "license": "MIT",
-  "main": "dist/ip-address.js",
-  "types": "dist/ip-address.d.ts",
+  "main": "dist/cjs/ip-address.js",
+  "module": "dist/esm/ip-address.js",
```

Comment:
looks like we lost the `types` declaration here?

Reply:
Fixed with de9c6c2